core/summary_api.php

```diff
@@ -116,126 +121,98 @@ function summary_print_by_enum( $p_enum ) {
 				ORDER BY ' . $p_enum . ' ' . $t_status_query;
 	$t_result = db_query( $t_query );
 
-	$t_last_value = -1;
+	$t_resolved_val = config_get( 'bug_resolved_status_threshold' );
+	$t_closed_val = config_get( 'bug_closed_status_threshold' );
+
+	$p_cache = array();
```

Comment:
`p_` prefix is for function parameters, I guess you meant `t_cache` ?

Reply:
got it.